api/room/schema_query.py

```diff
@@ -140,6 +152,25 @@ def resolve_all_rooms(self, info, **kwargs):
         response = PaginatedRooms(**kwargs)
         return response
 
+    def resolve_all_remote_rooms(self, info):
```

Comment:
According to your tests, seems only the admin should be able to get the list of rooms so maybe you should include `@Auth.user_roles('Admin')` before your function. Unless anybody is allowed to access in which case maybe the test should be refactored

Reply:
Thanks @michael-basweti for the feedback, I just asked @joshuaocero and he said only admins should be able to do this, so I will add the decorator to the method.

Reply:
> Thanks @michael-basweti for the feedback, I just asked @joshuaocero and he said only admins should be able to do this, so I will add the decorator to the method.
@kampkelly , that will work

Reply:
Great, it has been done.